app/models/user.rb

```diff
@@ -3,7 +3,16 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  geocoded_by :zip
+
+  geocoded_by :zip do |user, results|
```

Comment:
@rickr - Not terribly pleased that this is a block being passed, at the top of the `user.rb` file, versus a method being called.  However, was not able to find an alternative solution in the [Geocoder docs](https://github.com/alexreisner/geocoder).  

Let me know if you know of one/want it changed.